gitax/Visual Studio Projects/C Sharp Projects/GTXLibGit2Sharp/GTXRepo.cs

```diff
@@ -20,8 +20,8 @@ public static class GTXRepo
         /// <param name="repoPath">the repository main path</param>
         public static void Init(string repoPath)
         {
-            if (!Repository.IsValid(repoPath))
-                Repository.Init(repoPath, false);
+            if (!Repository.IsValid(repoPath))            
+             Global.error(repoPath  + " is not a valid git repo");            
```

Comment:
I agree that a manual git init might be a bit ott. The way I would like to use gitax would not be standalone but in combination with http://www.sourcetreeapp.com/  which can perform all the git commands from a gui and give you all the visibility that you need. It's far easier to make a clean commit that way than doing it from AX.
Gitax in that scenario only has to focus on the actual integration/syncing and doesn't have to support all the git options.
Of course, I'm completely biased here towards our own use case! :)